src/openfl/display/SimpleButton.hx

```diff
@@ -101,6 +101,19 @@ class SimpleButton extends InteractiveObject {
 	}
 	
 	
+	private override function __forceRenderDirty ():Void {
+		
+		super.__forceRenderDirty();
+		
+		if (__currentState != null) {
+			
+			__currentState.__forceRenderDirty ();
+			
+		}
```

Comment:
As discussed, the `hitTestState` doesn't need to be set to dirty. It is not rendered and thus not effected by something like `devicePixelRatio`. Nice `;)`